inference-engine/tests/functional/plugin/shared/src/single_layer_tests/trivial_concat.cpp

```diff
@@ -37,34 +37,42 @@ namespace LayerTestsDefinitions {
         return result.str();
     }
 
-    void Concat4DLayerTest::SetUp() {
-        int axis = 1;
+    void TrivialConcatLayerTest::SetUp() {
         InferenceEngine::SizeVector inputShape;
         InferenceEngine::Precision netPrecision;
         std::map<std::string, std::string> additional_config;
         std::tie(inputShape, netPrecision, targetDevice, additional_config) = this->GetParam();
         configuration.insert(additional_config.begin(), additional_config.end());
-
-        auto total_size = std::accumulate(inputShape.begin(), inputShape.end(), static_cast<size_t>(1), std::multiplies<size_t>());
+        int axis = inputShape.size() - 2;
+        size_t total_size = std::accumulate(inputShape.begin(), inputShape.end(), static_cast<size_t>(1), std::multiplies<size_t>());
         auto ngPrc = FuncTestUtils::PrecisionUtils::convertIE2nGraphPrc(netPrecision);
-        auto params = ngraph::builder::makeParams(ngPrc, {inputShape});
-        auto input = params[0];
+        auto params = ngraph::builder::makeParams(ngPrc, {{1, total_size}});
+
+        auto input_relu = ngraph::builder::makeActivation(params[0], ngPrc, ngraph::helpers::ActivationTypes::Relu);
```

Comment:
More over, it looks like subgraph test

Reply:
GNA cannot test with just Concat since then it is trivial (no gna operations) so activations are added to make it viable for GNA plugin, while RELU is selected since it does almost nothing.

Reply:
Anyway if this is moved to subgraph as act_trivialConcat_act then will it be fine for you to have this test there?